policyengine/tests/economic_impact/winners_and_losers/test_by_income_decile/by_income_decile.yaml

```diff
@@ -0,0 +1,19 @@
+# Winners and Losers by income
```

Comment:
Can we change this to a US test?

Reply:
Is there any specific parameter you would recommend that could impact our target metrics ?

Reply:
This one- see `reproduce in python` for the parameter name. https://policyengine.org/us/policy?reform=51488&focus=gov.irs.credits.ctc.refundable.fully_refundable&region=us&timePeriod=2024&baseline=2

Reply:
updated with the requested parameter, although we still have zeroes at some places.